### PR TITLE
Issue/26 remove sbt bintray bundle

### LINKED ISSUE #26: Bintray credentials are required to run the app
After [the pull request that added bintray publishing](https://github.com/lagom/activator-lagom-java-chirper/pull/14) simply using the `runAll` command will fail if you do not have a bintray credentials file set up. This is an issue at root with sbt-bintray. However, this should be removed from here until it is resolved at the root.

The current workaround is to create the credentials file, or remove the lines added in the referenced PR.

### PR DESCRIPTION
**Pull request by dotta**
Fix #26 \cc @huntc

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.0.0-RC1")
-addSbtPlugin("com.typesafe.sbt" % "sbt-bintray-bundle" % "1.0.2")
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0")
 addSbtPlugin("com.github.ddispaltro" % "sbt-reactjs" % "0.5.2")
 addSbtPlugin("com.lightbend.conductr" % "sbt-conductr" % "2.1.4")
```
Comment (huntc) [inline project/plugins.sbt:4]: Old plugin - should be 2.1.7

**Comment by huntc** [comment]
One comment, otherwise LGTM

File: project/plugins.sbt (hunk 1 above)
Comment (dotta) [inline project/plugins.sbt:4]: It's 2.1.7 in [this PR](https://github.com/lagom/activator-lagom-java-chirper/pull/20/files#diff-a84f91f20f040218bccd09fed4761fb3R5) ;-)

**Comment by dotta** [comment]
thx for the quick turnaround!

**Pull request merged**